assembly/integer/safe/u128.ts

```diff
@@ -226,6 +226,12 @@ class u128 extends U128 {
     );
   }
 
+  // TODO: ensure saftey
```

Comment:
unsigned div already safe by default (already checked div by zero) so you could remove this comment